test_haystack/solr_tests/server/wait-for-solr

```diff
@@ -8,7 +10,10 @@ import requests
 max_retries = 100
 retry_count = 0
 retry_delay = 15
-status_url = "http://localhost:9001/solr/collection1/admin/ping"
+status_url = "http://%s:%s/solr/collection1/admin/ping" % (
+    os.environ.get("SOLR_HOST", "localhost"),
+    os.environ.get("SOLR_PORT", "9001"),
```

Comment:
Shouldn't we default to the default Solr port?
```suggestion
    os.environ.get("SOLR_PORT", "8983"),
```

Reply:
We should - just need to update that in every spot but the old port choices date back to when people were running the test daemons locally without Docker